Source/Tests/Paths/ConfigDirPathTests.cs

```diff
@@ -2,25 +2,36 @@ namespace CodeImp.Bloodmasters.Tests.Paths;
 
 public class ConfigDirPathTests
 {
-    [Fact(DisplayName = "ConfigDirPath should be named 'Debug' in dev mode and 'Config' otherwise")]
-    public void ConfigDirPathShouldBeNamedDebugInDevModeAndConfigOtherwise()
+    [Fact(DisplayName = "ConfigDirPath should be named 'Config' in production mode")]
+    public void ConfigDirPathShouldBeNamedConfigInProductionMode()
     {
         // Arrange
-        var dirName = Path.GetFileName(CodeImp.Bloodmasters.Paths.ConfigDirPath);
+        var sut = CodeImp.Bloodmasters.Paths.Instance;
+        var dirName = Path.GetFileName(sut.ConfigDirPath);
 
         // Assert
-        Assert.Equal(
-            CodeImp.Bloodmasters.Paths.IsDevModeBuild ? "Debug" : "Config",
-            dirName);
+        Assert.Equal("Config", dirName);
     }
 
-    [Fact(DisplayName = "ConfigDirPath should be subdirectory of the 'Bloodmasters' directory in dev mode")]
-    public void DownloadedResourceDirShouldBeSubdirectoryOfBloodmasters()
+    [Fact(DisplayName = "ConfigDirPath should be named 'Debug' in dev mode",
+        Skip = "Dev mode requires .bloodmasters.dev.marker file")]
```

Comment:
If I understand correctly, it only requires the file because you've added an assertion there, right?

If so, then let's consider a small refactoring: let's
- make the `DevPaths` class public
- introduce a new constructor parameter `bool shouldAssert = true`
- pass `shouldAssert: true` when a `DevPaths` instance is created from `Paths.Create`
- create a test instance as `new DevPaths(shouldAssert: false)` in tests

Will this make the tests work?

Reply:
No. There are two files to run in dev mode:
- `.bloodmasters.dev.marker`. Copied next to each game binary so that it can detect dev mode
- `.bloodmasters.solution-root.marker`. Used as a marker for the folder that is used as the root path when calculating other paths

These files are not present when the tests are executed. And this causes the tests to drop in dev mode

Reply:
Please, check https://github.com/ForNeVeR/bloodmasters/pull/149/commits/55853f2b4c1d46c95c4be81747dba663073dbc30